ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -362,6 +362,30 @@ interface Ethernet1
    l2 mru 8000
    bgp session tracker ST1
    no switchport
+   switchport tap native vlan 10
+   switchport tap identity 3 inner 5
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap mpls pop all
+   switchport tap encapsulation gre strip
+   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0000 strip
```

Comment:
In my device the running config sequence is different for tap mpls pop all. The other point is not all tap encapsulation commands reflecting in running config. Can you please check
```
interface Ethernet1
   switchport tap native vlan 10
   switchport tap identity 3 inner 5
   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
   switchport tap encapsulation gre strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x0 strip
   switchport tap mpls pop all
   switchport tool mpls pop all
```

Reply:
yes, the sequence for `tap mpls pop all` is getting disturbed becasuse I have to add  it with `eilf` for `vxlan_strip`

Reply:
ok can you add the comment in J2 for future reference the reason behind order miss-match .

Reply:
Added